brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3602 Fix: Global - Resolved an issue of static page selected as front-page styling would not load as intended in FSE themes.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3602

### Screenshots

- https://d.pr/v/Fluy05

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

#Test case 1
- Go to Editor > Template > Pages
- Add some spectra block and designed it.
- Now go to settings > Reading > Static page and select and random page and home page
- Visit the site url. Make sure style should not break for anything.

#Test case 2
- Create a custom template by going into Appearance > Editor > Template > Click on + icon > Scroll and click on custom template.
- Created a custom template with Spectra block.
- Save it, Go to page, create new page, Select Template as a name of custom template.
- And Publish the page and visit frontend, in master you will see everything is broken.
- Now, switch to the branch, Regenerate an assets and refresh the page. You will see broken issue is fixed and style loaded properly.

### Checklist:
